client/src/components/base/__tests__/formValidation.test.ts

```diff
@@ -88,6 +88,7 @@ describe("Form Validation", () => {
       );
     } catch (e) {
       const error = e as ValidationError;
+      console.log(error);
```

Comment:
Left over from testing?

Reply:
I need to implement that local Github action you shared 🫣